### PR TITLE
Added Inventory Item Model and Service

### PR DESCRIPTION
**Pull request by deepsidhu1313**
* Added Model and Service for Inventory Item (https://shopify.dev/api/admin-rest/2022-04/resources/inventoryitem).
* Added Unit test for the new service

### CODE REVIEW

File: src/Models/InventoryItem.php [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+use BoldApps\ShopifyToolkit\Contracts\Serializeable;
+use BoldApps\ShopifyToolkit\Traits\HasAttributesTrait;
+
+class InventoryItem implements Serializeable, \JsonSerializable
+{
+    use HasAttributesTrait;
+
+    /** @var string */
+    protected $cost;
+    /** @var string|null */
```
Comment (Wryte) [inline src/Models/InventoryItem.php:14]: super nit: spacing inconsistent here

**Review by Wryte — CHANGES_REQUESTED** [review]
Are the nullable properties on the model from the documentation? For the setters of those properties should they be able to take null as well?

File: src/Models/InventoryItem.php (hunk 1 above)
Comment (deepsidhu1313) [reply src/Models/InventoryItem.php:15]: is this better ?

**Comment by deepsidhu1313** [comment]
removed type hinting and updated code to accept null values in setters. I took the example given on shopify https://shopify.dev/api/admin-rest/2022-04/resources/inventoryitem to create this model and service

**Pull request merged**